transport/src/main/java/io/el/channel/DefaultChannelPromise.java

```diff
@@ -0,0 +1,85 @@
+package io.el.channel;
+
+import io.el.concurrent.Promise;
+import io.el.concurrent.PromiseListener;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+/**
+ * TODO: implement this
+ */
+public class DefaultChannelPromise implements ChannelPromise {
```

Comment:
Implementation of this class is not included in issue https://github.com/zeroFruit/el/pull/37